contracts/proxy-lib/src/lib.rs

```diff
@@ -56,6 +57,7 @@ impl ProxyContract {
             num_approvals: 3,
             active_proposals_limit: 10,
             context_storage: IterableMap::new(b"l"),
+            code_size: env::storage_usage() as u64,
```

Comment:
ah, at initialization is `storage_usage` == `code size`? I'd think it shouldn't be, right? Should vary by a few bytes, but yeah this is fine.